core/src/main/java/cz/xtf/core/openshift/OpenShiftBinary.java

```diff
@@ -52,7 +53,15 @@ public String execute(String... args) {
         if (ocConfigPath == null) {
             return executeCommand(ArrayUtils.addAll(new String[] { path }, args));
         } else {
-            return executeCommand(ArrayUtils.addAll(new String[] { path, "--config=" + ocConfigPath }, args));
+            log.debug("openshift version: [{}]", OpenShifts.getVersion());
+            if (StringUtils.isNotEmpty(OpenShifts.getVersion())) {
+                String[] ocpVersion = OpenShifts.getVersion().split("\\.");
+                if (Integer.parseInt(ocpVersion[0]) < 4
+                        || Integer.parseInt(ocpVersion[0]) == 4 && Integer.parseInt(ocpVersion[1]) < 6) {
```

Comment:
To keep 4.6 behavior unchanged the last condition should be `< 7` or `<= 6`. Not a big problem as `--kubeconfig` works with 4.6.

Reply:
should I change the condition then or it is fine like that?
I mean for OCP 4.6 the `--kubeconfig` is working as well as `--config`

Reply:
It is fine for me, but let's wait for @mnovak1 (OOO today) or any other reviewer. I cannot merge it anyway.